ngraph/core/src/itt.hpp

```diff
@@ -49,6 +49,13 @@ OV_ITT_DOMAIN(SIMPLE_ngraph_pass);
         }                                                       \
     } break
 
+#define NGRAPH_TYPE_CASE2(region, a, b, ...)                                    \
```

Comment:
Maybe can we imagine a more clear name (without 2 suffix)?

Reply:
I've changed it to `NGRAPH_2_TYPES_CASE`.